docs/installing.md

```diff
@@ -1,104 +1,58 @@
 
-## 1. Download dependencies
+## 1. Install dependencies
 
-1. Install git, git-lfs, and Java. On Ubuntu: `sudo apt install git git-lfs openjdk-11-jdk`
-2. Download the Monero binaries. To make things easier, we provide two custom Monero binaries: `monerod` and `monero-wallet-rpc`. You can directly [download the software (Linux and Mac)](https://github.com/haveno-dex/monero/releases/tag/testing) or build the binaries yourself by cloning and building the `release-v0.17` branch of this repository: https://github.com/haveno-dex/monero
-3. Download and install [Bitcoin-Qt](https://bitcoin.org/en/download).
+On Ubuntu: `sudo apt install make wget git git-lfs openjdk-11-jdk`. The Bitcoin and Monero binaries will be downloaded and verified automatically in the next step.
 
 ## 2. Build Haveno
 
-1. Download this repository. Run `git clone https://github.com/haveno-dex/haveno.git`
-2. Navigate inside the repository (`cd haveno`) and start the build: `./gradlew build`. It will take some minutes. If you know what you are doing you can modify [WalletConfig.java](/core/src/main/java/bisq/core/btc/setup/WalletConfig.java) with custom settings before building.
+1. Download this repository: `git clone https://github.com/haveno-dex/haveno.git`
+2. Navigate to the root of the repository (`cd haveno`) and build the repository: run `make`in the terminal and wait until the process is completed.
 
-## 3. Run Monero
+## 3. Bitcoin
 
-1. Unpack the downloaded or built archives from step 1.2 and copy `monero-wallet-rpc` to the root of this repository. Make sure it has the right permissions (if you are in doubt, just run `chmod 777 monero-wallet-rpc`).
-2. Now join our shared stagenet (skip this command and see the next section if you prefer to test Haveno using a local private stagenet). Navigate into the folder containing `monerod` and run:
+Run each of these commands in their own terminal window:
 
-```
-./monerod --stagenet --no-igd --hide-my-port --data-dir stagenet --add-exclusive-node 136.244.105.131:38080 --rpc-login superuser:abctesting123 --rpc-access-control-origins http://localhost:8080 --fixed-difficulty 10
-```
+1. Run Bitcoin: `make bitcoind`
+2. Create bitcoin blocks: `make btc-blocks` (after running the command, this terminal window can be closed)
 
-Your daemon will connect to our private Monero stagenet and will be ready to be used by Haveno.
+## 4. Connect to Monero stagenet
 
-### Run private stagenet
+The quickest way to get a Monero stagenet running is by connecting to our own shared instance (4a). You will connect to an already running stagenet maanged by us and you won't have to do anything beside mining some coins for testing (step 6). if you prefer to have total control over the testing instance, you might prefer running your own private Monero stagenet (4b).
```

Comment:
The quickest way to get a Monero stagenet running is by connecting to our own shared instance (4a) so you won't have to do anything except mine some coins for testing (step 6). If you prefer to have total control over the testing instance, you might prefer running your own private Monero stagenet (4b).